pkg/registry/apis/secret/contracts/secure_value.go

```diff
@@ -10,7 +10,8 @@ import (
 )
 
 var (
-	ErrSecureValueNotFound = errors.New("secure value not found")
+	ErrSecureValueNotFound      = errors.New("secure value not found")
+	ErrSecureValueAlreadyExists = errors.New("secure value already exists")
```

Comment:
We could probably have this in the future as a custom error struct that can be converted into a ` *apierrors.StatusError` 🤔 for easier handling in the reststorage